src/lib/extension.ts

```diff
@@ -1 +1,3 @@
 export const IDENTIFIER = 'aha-develop.gitlab';
+export const LEARN_MORE_URL = 'https://github.com/aha-develop/gitlab';
+export const ICON = aha.iconForExtensionIdentifier(IDENTIFIER);
```

Comment:
Does this PR depend on an unreleased change to the `aha` global? I'm seeing this exception in the endpoint handler logs:

```
TypeError: aha.iconForExtensionIdentifier is not a function
JavaScript at src/lib/extension.ts:3:25
JavaScript at aha-develop.gitlab.webhook:239:3
JavaScript at aha-develop.gitlab.webhook:240:3
JavaScript at Aha.defineExtension (ContributionRegistry.js:50:7)
JavaScript at aha-develop.gitlab.webhook:1:34
JavaScript at aha-develop.gitlab.webhook:241:4
```

Reply:
Oh just saw this in the PR message, duh: "Requires https://github.com/aha-app/aha-app/pull/23132 for the icon, but this is not a release blocker"

Based on the behavior in the endpoint contribution, I believe this is a blocker though: the handler fails to run due to this exception.

Reply:
ah, I understand. I need to not include that call in the endpoint

Reply:
@trydionel this should be fixed now